[rubocop] Autocorrect

donaldong opened the pull request.

donaldong commented:
update

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/4a0e1e1715552dd997e3d536394a99b6a6b9b694..8440870205f962cf14409c3bc12591c7d8425c28).

donaldong commented:
fix

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/1409125f8fc96fa0a45875f6b5380c89d829a042..3653b84ac8d4f3af0423d940abf539aa65a72142).

katyho commented on lib/redis_memo/redis.rb:
```diff
@@ -43,9 +44,9 @@ def initialize(orig_options)
-    alias_method :get_primary, :get
-    alias_method :mget_primary, :mget
-    alias_method :mapped_mget_primary, :mapped_mget
+    alias get_primary get
+    alias mget_primary mget
+    alias mapped_mget_primary mapped_mget
```
Is this a rubocop suggestion? I did a quick google search, and it feels like alias_method is a bit more straightforward to use/more predictable

donaldong replied:
Yes it is. I agree with you tho! Let me undo this change and adjust rubocop

katyho commented on lib/redis_memo/testing.rb:
```diff
@@ -5,13 +5,12 @@
-    def self.__test_mode
-      @__test_mode
+    class << self
+      attr_reader :__test_mode
```
Should we just use attr accessor here?

donaldong replied:
huh... thanks! good catch

donaldong commented on lib/redis_memo/memoizable/dependency.rb:
```diff
@@ -12,13 +12,14 @@ def memos
   end
 
-  def depends_on(dependency, **conditions)
+  def depends_on(dependency, **_conditions)
```
good point!

donaldong commented:
update

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/07312cdd357dca139213dc472067dd72b59da970..cd55c9971d978d93866e3ffd0cb1182793b2ed50).

donaldong commented:
fix spec

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/5bff904a4a27d91579d9a661c4270b7ce914b69c..c68072d2d8ffa9549f3e42b0a4518ca0377c9f5c).
